handlers.go

```diff
@@ -17,8 +18,46 @@ func newSpecialReportsHandler(service specialReportService) specialReportsHandle
 	return specialReportsHandler{service: service}
 }
 
+func (h *specialReportsHandler) count(writer http.ResponseWriter, req *http.Request) {
+	ids := h.service.getSpecialReportIds()
+	_, err := writer.Write([]byte(strconv.Itoa(len(ids))))
+	if err != nil {
+		log.Warnf("Couldn't write count to HTTP response. count=%d %v\n", len(ids), err)
+		writer.WriteHeader(http.StatusInternalServerError)
+	}
+}
+
+func (h *specialReportsHandler) ids(writer http.ResponseWriter, req *http.Request) {
+	ids := h.service.getSpecialReportIds()
+	writer.Header().Add("Content-Type", "text/plain")
+	if len(ids) == 0 {
+		writer.WriteHeader(http.StatusOK)
```

Comment:
__ids and __count goes hand in hand, if __count returns 0, for me __ids returning an empty response does not suggest a failure. So just go ahead.

Note: the type of the body is basically a json stream, which by convention is just `application/json`. That could be updated.